packages/nimble-components/src/table/specs/table-column-specs/table-column-anchor-hld.md

```diff
@@ -0,0 +1,121 @@
+# Anchor Table Column HLD
+
+## Problem Statement
+
+We need to support links in table cells. We will do this by creating a new column type.
+
+## Links To Relevant Work Items and Reference Material
+
+[GitHub issue: Table allows configuration of hyperlink column type (#1012)](https://github.com/ni/nimble/issues/1012)
+
+## Implementation / Design
+
+This is a fairly straightforward execution of our custom column type pattern. The anchor column will use two fields from the data records: one for the link's visible text (the "label") and one for the url (the "href"). The column configuration will contain the rest of the native anchor parameters: `hreflang`, `ping`, `referrerpolicy`, `rel`, `target`, `type`, and `download`. The assumption is that these should be the same for each link of a given column (if used at all). An alternative would be to have a `*FieldName` property for each of the native anchor parameters, but that seems unlikely to be useful.
+
+The column will also take an optional `placeholder` value to use when a record does not define a label.
```

Comment:
What will the behavior be if the record doesn't define the `href`?

Reply:
The template a little further down specifies that we render a text span instead of a link when the `href` value is missing, however I'll just add a note here about that as well.